src/bxml/startStream.ts

```diff
@@ -0,0 +1,78 @@
+import { XMLElement } from 'xmlbuilder'
+import { Verb } from './Verb'
+
+export interface StartStreamOptions {
+    destination: string
+
+    name?: string
+
+    tracks?: string
+
+    streamEventUrl?: string
+    
+    streamEventMethod?: string
+    
+    username?: string
+    
+    password?: string
+}
+
+export class StartStream implements StartStreamOptions, Verb {
+    destination: string
```

Comment:
If `destination` is required, we are not throwing any error from constructor. Not sure if the auto-generated code has bug or something we missed in the spec!